src/backend/marsha/core/api/account.py

```diff
@@ -95,3 +97,13 @@ def get_permissions(self):
             except AttributeError:
                 permission_classes = self.permission_classes
         return [permission() for permission in permission_classes]
+
+    def get_queryset(self):
+        """
+        override get_queryset based on the requestion action
+        """
+
+        if self.action in ["list"]:
+            return super().get_queryset().filter(users__id=self.request.user.id)
```

Comment:
To respect PR purpose "Users can list organization they are administrator for" I think a filter is lacking:
```suggestion
            from marsha.core.models import OrganizationAccess
            from marsha.core.models import ADMINISTRATOR
            
            return super().get_queryset().filter(
                user_accesses__user_id=self.request.user.id,
                user_accesses__role=ADMINISTRATOR,
            )
```

Reply:
Actually to be safer, I would also use the same filter for `if self.action in ["retrieve", "update", "partial_update"]:`

Reply:
Yes definitely !